DOC silverstripe/widgets removed (#283)
silverstripe/developer-docs: en/04_Changelogs/5.1.0.md

- [`BuildTask`](api:SilverStripe\Dev\BuildTask) now has boolean `is_enabled` configuration option which has precedence over the existing `BuildTask::enabled` protected class property. The `BuildTask::enabled` property has been marked as deprecated and will be removed in CMS 6 if favour of using `is_enabled` instead.

## Dependency changes

> - The unsupported module `silverstripe/widgets` was removed from `silverstripe/recipe-blog` after it was accidentally included in the `2.0.0` release of `silverstripe/recipe-blog`. If your project relies on `silverstripe/widgets` which is CMS 5 compatible though unsupported, you can still it to your project manually in your `composer.json` file.

Reviewer: We should mentioned both modules. Also, try to use simpler/shorter sentences where possible.

```suggestion
- The unsupported modules `silverstripe/widgets` and `silverstripe/content-widget` were removed from `silverstripe/recipe-blog`. They were accidentally included in the `2.0.0` release of `silverstripe/recipe-blog`. The `silverstripe/widgets` and `silverstripe/content-widget` modules are CMS-5-compatible though unsupported. If your project relies on `silverstripe/widgets` or `silverstripe/content-widget`, manually update your project's `composer.json` file to explicitly require these modules.
```

Author: Updated
